src/main/java/hudson/tasks/junit/SuiteResult.java

```diff
@@ -347,20 +318,9 @@ public float getDuration() {
     }
 
     /**
-     * The possibly-null {@link Run#getExternalizableId()} this suite was generated in.
-     *
-     * @since 1.21
-     */
-    @CheckForNull
-    @Exported(visibility=9)
-    public String getRunId() {
-        return runId;
-    }
-
-    /**
      * The possibly-null {@link FlowNode#id} this suite was generated in.
      *
-     * @since 1.21
+     * @since 1.22
```

Comment:
BTW I really do not recommend merging API stuff into `master` until the whole set of related changes has been fully implemented and approved.